### PR TITLE
MSSQL: Add session timezone

### PR DESCRIPTION
**Pull request by alyssajoyner**
This PR fixes an issue where the server's timezone is not in UTC. It adds a setting in `Additional Settings` on the MSSQL Configuration page to allow a user to input a timezone so that dashboards can be loaded correctly.

Screenshot 2024-01-31 at 9 11 09 PM

### CODE REVIEW

**Review by lwandz13 — APPROVED** [review]
Looks good!

File: public/app/plugins/datasource/mssql/configuration/ConfigurationEditor.tsx [hunk 1]
```diff
@@ -295,6 +299,36 @@ export const ConfigurationEditor = (props: DataSourcePluginOptionsEditorProps<Ms
...
+              <Stack gap={0.5}>
+                <span>Session timezone</span>
+                <Tooltip
+                  content={
+                    <span>
+                      Specify the timezone used in the database session, such as <code>Europe/Berlin</code> or
+                      <code>+02:00</code>. Required if the timezone of the database (or the host of the database) is set
+                      to something other than UTC. Set this to <code>+00:00</code> so Grafana can handle times properly.
+                      Set the value used in the session with <code>SET time_zone=&apos;...&apos;</code>. If you leave
+                      this field empty, the timezone will not be updated. You can find more information in the MySQL
```
Comment (codeincarnate) [inline public/app/plugins/datasource/mssql/configuration/ConfigurationEditor.tsx:314]: Ooh ooh, I've worked on this. Just a quick catch `MySQL` --> `MSSQL` :smile:

File: public/app/plugins/datasource/mssql/configuration/ConfigurationEditor.tsx [hunk 2]
```diff
@@ -295,6 +299,36 @@ export const ConfigurationEditor = (props: DataSourcePluginOptionsEditorProps<Ms
...
+                </Tooltip>
+              </Stack>
+            </Label>
+          }
+        >
+          <Input
+            width={40}
+            value={jsonData.timezone || ''}
+            onChange={onUpdateDatasourceJsonDataOption(props, 'timezone')}
+            placeholder="Europe/Berlin or +02:00"
```
Comment (codeincarnate) [inline public/app/plugins/datasource/mssql/configuration/ConfigurationEditor.tsx:329]: for the placeholder might be good to put `e.g.` before the rest of the text

**Review by codeincarnate — APPROVED** [review]
Other than little nits looks good 😄

**Comment by alyssabull** [comment]
Closing this PR as Grafana is designed to work with databases utilizing UTC time. We will not be fixing this issue as of now.

**Pull request closed without merging**